engine/src/main/java/org/destinationsol/game/SolGame.java

```diff
@@ -458,7 +426,7 @@ public void drawDebug(GameDrawer drawer, Context context) {
 
     private void drawDebugPoint(GameDrawer drawer, Vector2 dp, Color col) {
         if (dp.x != 0 || dp.y != 0) {
-            float sz = context.get(SolCam.class).getRealLineWidth() * 5;
+            float sz = getContext().get(SolCam.class).getRealLineWidth() * 5;
```

Comment:
This is debug code, so it isn't running most of the time. Since this just replaces the existing code with a functional replacement, nothing needs to change here.